trino/trino.go

```diff
@@ -362,9 +366,14 @@ func newConn(dsn string) (*Conn, error) {
 		trinoSchemaHeader:          query.Get("schema"),
 		trinoSessionHeader:         query.Get("session_properties"),
 		trinoExtraCredentialHeader: query.Get("extra_credentials"),
+		authorization:              query.Get("bearer_auth"),
```

Comment:
Add a method that would prefix the value of `query.Get("bearer_auth")` if it's not empty to avoid the condition in the loop below.

Reply:
@nineinchnick - Fixed. Please check.